src/css/custom.scss

```
@@ -556,3 +556,98 @@ body .DocSearch {
     var(--ifm-color-emphasis-100) 100%
   );
 }
+
+/* Markdown Actions Dropdown Styles */
+
+/* Style the article header as flexbox */
+article .markdown header {
+  display: flex;
+  align-items: center;
+  /*flex-wrap: wrap;*/
```

Stylelint (17.9.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)